### ISSUE TITLE
Skeleton.Group is broken

### ISSUE THREAD

**Issue by peterlazar1993** [nandorojo/moti#275]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

`Group` appears to be not present on `Skeleton`.

So on Rendering `Skeleton.Group`, React throws this error

```
Error: Element type is invalid: expected a string (for built-in components) or a class/function
(for composite components) but got: undefined. You likely forgot to export your component
from the file it's defined in, or you might have mixed up default and named imports.
```

### Expected Behavior

`Skeleton.Group`  is available.

### Steps To Reproduce

1. import { Skeleton } from 'moti/skeleton';
2. Render `Skeleton.Group`

### Versions

```markdown
- Moti: 0.24.1
- Reanimated: 3.0.1
- React Native: 0.71.3
```

### Screenshots

<img src="https://user-images.githubusercontent.com/4848554/223428774-9d61f4bd-d2ab-441f-8367-dc04634c1575.png" width=40% >

### Reproduction

https://github.com/peterlazar1993/skeleton-bug/blob/main/App.js

**Comment by nandorojo** [comment]
Ah shoot, will be an easy fix.

**Comment by nandorojo** [comment]
Fixed in `0.24.2`
